includes/forms.inc

```diff
@@ -448,7 +448,7 @@ function islandora_ip_embargo_object_embargo_form($form, &$form_state, $islandor
   $form_state['storage']['islandroa_ip_embargo']['update'] = FALSE;
   $form_state['storage']['islandroa_ip_embargo']['pid'] = $islandora_object->id;
   $list_results = islandora_ip_embargo_get_lists();
-  $options = array('none' => NULL);
```

Comment:
The 'none' list item is [intended to be used below](https://github.com/Islandora-Labs/islandora_ip_embargo/blob/6761ba3eb9930c229a43dffdbee8f3b576898182/includes/forms.inc#L520) as a way of removing IP embargo lists when updating ... seems like the problem is more the fact that you can try to create a new list using a list ID of 'none' and it just happily tries to add that to the database.

My recommendation would be:
* Give the 'none' list option an actual label (e.g., `array('none' => t('No IP Embargo'))` so it's much more clear what it's trying to do
* Remove the `#required` designation
* Create an `islandora_ip_embargo_object_embargo_form_validate()` handler that ensures `$form_state['values']['lists']` is not 'none' when `$form_state['storage']['islandroa_ip_embargo']['update'] == TRUE`